test/runtests.jl

```diff
@@ -23,22 +16,28 @@ using Random
 using StaticArrays
 using Statistics
 using Test
+using JuliaInterpreter
 
-@nospecialize
-
-if isdefined(Base, :Experimental) && isdefined(Base.Experimental, Symbol("@compiler_options"))
-    @eval Base.Experimental.@compiler_options compile=min optimize=0 infer=no
-end
+union!(JuliaInterpreter.compiled_modules, Any[Base, Base.Broadcast, Compat, LinearAlgebra, Random, StaticArrays, Statistics])
```

Comment:
comment this line?

Reply:
What if we run FiniteDifferences.jl in compiled?

Reply:
OK, I will.

> What if we run FiniteDifferences.jl in compiled?

I should AB-test that again, but I think a lot of the improvements here are coming from not compiling FiniteDifferences.

Reply:
ah, but we *do* compile Base, which contains the primal functions that are actually being called by FiniteDifferences.jl
I get it.
Makes sense

Reply:
Just checked again to be sure and compiling FiniteDifferences is indeed slower.